tests/NHapi.NUnit/NHapi.NUnit.csproj

```diff
@@ -58,12 +58,8 @@
     <ProjectReference Include="..\..\src\NHapi.Model.V28\NHapi.Model.V28.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
-    <PackageReference Include="SimpleSpeedTester" Version="1.1.0" />
```

Comment:
Yes, these did simple performance testing in w of the unit tests, I have moved the performance test aspect of the test to the new benchmark project instead. Especially since that library only supported netframework, actually that reminds me I can probably remove the `#IF Net462` preprocessor symbols from around the test fixture now.

Reply:
@AMCN41R  okay changes made